lib/src/layout/toolbar/toolbar_overflow_menu.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:ui';
+
+import 'package:macos_ui/src/library.dart';
+import 'package:macos_ui/src/theme/macos_theme.dart';
+
+const BorderRadius _kBorderRadius = BorderRadius.all(Radius.circular(5.0));
+
+/// A menu that includes all overflowed toolbar actions.
+class ToolbarOverflowMenu extends StatelessWidget {
+  /// Builds a menu that includes all overflowed toolbar actions and appears
+  /// as a popup below the [ToolbarOverflowButton].
+  ///
+  /// Has a similar styling to a pulldown menu.
+  const ToolbarOverflowMenu({
+    Key? key,
+    required this.children,
+  }) : super(key: key);
+
+  /// The list of children widgets to lay out vertically inside the menu.
+  final List<Widget> children;
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
Are you sure about the use of `CupertinoColors` rather than `MacosColors` in this widget?

Reply:
You are right. I copied this from the pulldown and popup button implementations. Since there should be `MacosColors` alternatives, we should replace these.

Reply:
Let's actually leave `CupertinoColors` for now, we don't have equivalents for some of what you're using here.